CI/mxe/before_install.sh

```diff
@@ -3,34 +3,36 @@
 # Install nsis for installer creation
 sudo apt-get install -qq nsis ninja-build
 
-# MXE repository was too slow for Travis far too often
-wget https://github.com/vcmi/vcmi-deps-mxe/releases/download/2018-02-10/mxe-$MXE_TARGET-2018-02-10.tar
-tar -xvf mxe-$MXE_TARGET-2018-02-10.tar
-sudo dpkg -i mxe-*.deb
-sudo apt-get install -f --yes
-
-if false; then
-# Add MXE repository and key
-echo "deb http://pkg.mxe.cc/repos/apt/debian wheezy main" \
-    | sudo tee /etc/apt/sources.list.d/mxeapt.list
-
-sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D43A795B73B16ABE9643FE1AFD8FFF16DB45C6AB
-
-# Install needed packages
-sudo apt-get update -qq
-
-sudo apt-get install -q --yes \
-mxe-$MXE_TARGET-gcc \
-mxe-$MXE_TARGET-boost \
-mxe-$MXE_TARGET-zlib \
-mxe-$MXE_TARGET-sdl2 \
-mxe-$MXE_TARGET-sdl2-gfx \
-mxe-$MXE_TARGET-sdl2-image \
-mxe-$MXE_TARGET-sdl2-mixer \
-mxe-$MXE_TARGET-sdl2-ttf \
-mxe-$MXE_TARGET-ffmpeg \
-mxe-$MXE_TARGET-qt \
-mxe-$MXE_TARGET-qtbase
+if true; then
+	# MXE repository was too slow for Travis far too often
+	wget https://github.com/nullkiller/vcmi-deps-mxe/releases/download/2021-01-22/mxe-$MXE_TARGET-2021-01-22.tar
```

Comment:
This should be pushed to main repo

Reply:
It needs to be released. And I have no access to the official repo to make a release or at least push a tag so that I can try github-release utility. You can check that in my repo my commit is tagged and a release is created. Was not able to push a tag using a token provided in vcmi-deps-windows.

Reply:
Published releases for mxe and vcpkg